frame/support/procedural/src/pallet_version.rs

```diff
@@ -0,0 +1,64 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2020 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! Implementation of macros related to pallet versioning.
+
+use proc_macro2::{TokenStream, Span};
+use syn::{Result, Error};
+use std::{env, str::FromStr};
+use frame_support_procedural_tools::generate_crate_access_2018;
+
+/// Get the version from the given version environment variable.
+///
+/// The version is parsed into the requested destination type.
+fn get_version<T: FromStr>(version_env: &str) -> std::result::Result<T, ()> {
```

Comment:
very minor but this function is capable of reading _any_ environment variable, not just version, thus could us a better name.

Reply:
What you mean by any here? Why should it read any?